docs/config.md

```diff
@@ -279,7 +279,7 @@ Data is obtained at the face centers, then interpolated to face nodes. Here, we
 It is important to notice that the target data location is again the face center mesh of the coupling partner. In the standard CHT case, where both data sets are exchanged by a nearest-projection mapping, this leads to two interface meshes (centers and nodes) per participant. Having both the centers and nodes defined, we can skip one interpolation step and read data directly to the centers (cf. picture solver B).
 
 {% note %}
-As already mentioned, the `Fluid` participant does not need to provide the mesh connectivity in case of a standard FSI. Therefore, the `Solid` participant needs to provide it and nothing special needs to be considered compared to other mapping methods. This implementation supports all CHT-related fields, which are mapped with a `consistent` constraint.
+As already mentioned, the `Fluid` participant does not need to provide the mesh connectivity in case of a standard FSI writing forces. It can, however, provide the mesh connectivity when writing stresses. Alternatively, the `Solid` participant can provide mesh connectivity on the displacements mesh, in which case nothing special needs to be considered in the adapter implementation compared to other mapping methods. This implementation supports all CHT-related fields, which are mapped with a `consistent` constraint.
```

Comment:
No unfortunately not. We can provide the connectivity, but writing the data (e.g. `Stress`) won't be handled correctly. That's one of (my) bad design decisions, which requires to address this in each data field again. Right now, I can only think of scenarios where this behavior would either fallback to NN (such as for `Forces` etc.) and be ignored or would just be erroneous. Enabling it doesn't make sense in my opinion, unless we change the implementation.

Reply:
How would you phrase this part of the documentation, then?

Reply:
Correct would be: Mapping with connectivity information is not implemented for FSI and therefore, not supported. It's only supported for CHT related fields.

Reply:
I updated the message and made it trigger an error again.